.github/workflows/build.yml

```diff
@@ -38,6 +38,17 @@ jobs:
           distribution: 'zulu'
       - name: Verify gradle-wrapper checksum
         uses: gradle/wrapper-validation-action@v1
+      - name: install macOS software
+        if: matrix.os == 'macos-latest'
+        run: |
+          brew install make clang-format
+          # Prevent Gradle from picking up Java 8 on Github CI runner.
+          echo "org.gradle.java.home=$JAVA_HOME_17_X64" > contiki-ng/tools/cooja/gradle.properties
```

Comment:
Instead of creating and git ignore a properties file inside the Cooja source tree - what do you think about adding it in the home directory instead as `~/.gradle/gradle.properties` ?

Also, why Java 17 and not Java 21?

Reply:
Updated to use `$HOME/.gradle/gradle.properties`.

Gradle 8.4 does not run on Java 21. Github macOS CI runners have Java 8, 11 and 17 preinstalled.